### PR TITLE
update qc section of recipe

### PR DESCRIPTION
**Pull request by ErinWeisbart**
Updates QC section including:

Major changes:
- Plot empty cells per well separately from cell quality ratios per well
- Overhaul and simplification of correlation metrics to single by well plot
- Changing how some column names are determined so that they are filtered from lists of actual column names

Minor changes:
- Make wells square on by-well plots to better mimic whole-well-stitched images
- Changes to plot text aesthetics for readability
- More descriptive plot labels

### CODE REVIEW

File: 0.preprocess-sites/4.image-and-segmentation-qc.py
Comment (gwaygenomics) [inline 0.preprocess-sites/4.image-and-segmentation-qc.py:169]: I am not sure how else you are using `empty_df`, but if possible, you should try to keep data processing of a single variable all in one spot. What I mean is move this line closer to where you define `empty_df` (line 155).

This will help reader understanding a lot.

File: 0.preprocess-sites/4.image-and-segmentation-qc.py
Comment (gwaygenomics) [inline 0.preprocess-sites/4.image-and-segmentation-qc.py:182]: please make sure to run `black` on this script.

(Black will make sure there are no spaces in `ncol=3` 😄  )

File: 0.preprocess-sites/4.image-and-segmentation-qc.py
Comment (gwaygenomics) [inline 0.preprocess-sites/4.image-and-segmentation-qc.py:188]: black will fix this too

File: 0.preprocess-sites/4.image-and-segmentation-qc.py
Comment (gwaygenomics) [inline 0.preprocess-sites/4.image-and-segmentation-qc.py:295]: Just confirming that you no longer create a figure for all `painting_image_names`. Instead you are creating a single figure, correct?

**Review by gwaygenomics — APPROVED** [review]
Looks great to me. I made only minor comments that probably could use addressing before you merge. Thanks!

File: 0.preprocess-sites/4.image-and-segmentation-qc.py
Comment (ErinWeisbart) [reply 0.preprocess-sites/4.image-and-segmentation-qc.py]: hahaha. I can't even count the number of times I told myself "run `black` before making the PR" and yet I still forgot!

File: 0.preprocess-sites/4.image-and-segmentation-qc.py
Comment (ErinWeisbart) [reply 0.preprocess-sites/4.image-and-segmentation-qc.py:295]: I don't think `painting_image_names` was ever used to make separate figures? Yes, this is a single figure for all the painting images.

**Comment by ErinWeisbart** [comment]
@gwaygenomics Did a little more cleanup.
- addressed your comments
- fixed error in how percent empty figure was plotting and saving
- made similar changes to saturation plot as other plots to change column name determination
- saturation plot: made channels categorical and reordered them so that the wrap keeps A, C, G, T in columns (moving DAPI to end)

**Comment by gwaygenomics** [comment]
very nice - please merge when you are ready. (I assume you have write access)

**Pull request merged**
